### PR TITLE
[auto-trainer] Add labels param to log with the model

### PR DESCRIPTION
**Pull request by yonishelach**

### CODE REVIEW

File: auto_trainer/auto_trainer.py [hunk 1]
```diff
@@ -119,6 +119,7 @@ def train(
     test_set: DataItem = None,
     train_test_split_size: float = None,
     random_state: int = None,
+    labels: Dict[str, Union[str, int, float]] = None,
```
Comment (guy1992l) [inline auto_trainer/auto_trainer.py:122]:
```suggestion
    labels: dict = None,
```

I think its obvious what is a label dictionary :)

**Review by guy1992l — APPROVED** [review]
minor change, other than that, excellent :)
